RequestManagementSystem/private/RequestTask.py

```diff
@@ -158,13 +158,11 @@ def setupProxy( self ):
       return S_OK( { "Shifter": isShifter, "ProxyFile": proxyFile } )
 
     # # if we're here owner is not a shifter at all
-    ownerProxy = gProxyManager.downloadVOMSProxy( ownerDN, ownerGroup )
-    if not ownerProxy["OK"] or not ownerProxy["Value"]:
-      reason = ownerProxy["Message"] if "Message" in ownerProxy else "No valid proxy found in ProxyManager."
+    ownerProxyFile = gProxyManager.downloadVOMSProxyToFile( ownerDN, ownerGroup )
+    if not ownerProxyFile["OK"] or not ownerProxyFile["Value"]:
+      reason = ownerProxyFile["Message"] if "Message" in ownerProxyFile else "No valid proxy found in ProxyManager."
```

Comment:
`reason = ownerProxyFile.get('Message', "No valid proxy found in ProxyManager.")`